lib/hooks/core/hook-http.js

```diff
@@ -54,19 +53,21 @@ function requestWrap(request) {
       return request.apply(this, arguments);
     }
 
+    var localOptions = (typeof options === 'string') ?
```

Comment:
Since `options` are not referenced below this line; just assign to `options`. That way we cannot have an errant use of the original `options` object below this line.